tests/utilities/test_reader.py

```diff
@@ -384,22 +384,47 @@ def test_multiblockplot3dreader():
     # Reader doesn't yet support reusability
     reader = pyvista.MultiBlockPlot3DReader(filename)
     reader.add_q_files(q_filename)
+    reader.add_function(112)  # add by int
+    reader.add_function(pyvista.reader.Plot3DFunctionEnum.PRESSURE_GRADIENT)  # add by enum
     mesh = reader.read()
     for m in mesh:
         assert len(m.array_names) > 0
 
+    assert 'MachNumber' in mesh[0].point_data
+    assert 'PressureGradient' in mesh[0].point_data
+
```

Comment:
Can we add tests for the remove functionality by int and Enum?

Reply:
Good catch, now added!